### ISSUE TITLE
Error: aws_appsync_graphql.api: : invalid or unknown key: tags

### ISSUE THREAD

**Issue by pawanyoda** [terraform-providers/terraform-provider-aws#8472]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

*Recently launched aws. AWS AppSync Now Supports Tagging GraphQL APIs. I have created terraform script for but its throwing error.

### Terraform Version
`0.11.13`
### AWS Provider version
*aws: version = "~> 1.60"
### Affected Resource(s)

* aws_appsync_graphql_api

### Terraform Configuration Files

```hcl
resource "aws_appsync_graphql_api" "api" {
  authentication_type = "API_KEY"
  name                = "apisync-test"
  tags {
    Name   = "dev-test"
  }
}
```

### Panic Output
`Error: aws_appsync_graphql.api: : invalid or unknown key: tags`

**Comment by bflad** [comment]
Support for the `tags` argument has been speedily added by @acburdine and merged. It will be available in version 2.10.0 of the Terraform AWS Provider likely tomorrow. 👍

**Comment by bflad** [comment]
This has been released in [version 2.10.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2100-may-10-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
